assets/components/openshift-router/deployment.yaml

```
@@ -102,6 +102,34 @@ spec:
             - name: ROUTER_MUTUAL_TLS_AUTH_FILTER
               value: '{{ .ClientAuthFilter }}'
             {{- end }}
+            {{- if .AccessLoggingEnabled }}
+            - name: ROUTER_SYSLOG_ADDRESS
+              value: /var/lib/rsyslog/rsyslog.sock
+            - name: ROUTER_LOG_LEVEL
+              value: info
+            {{- if .HttpLogFormat }}
+            - name: ROUTER_SYSLOG_FORMAT
+              value: '{{ .HttpLogFormat }}'
+            {{- end }}
+            {{- if .HttpCaptureHeadersRequest }}
+            - name: ROUTER_CAPTURE_HTTP_REQUEST_HEADERS
+              value: '{{ .HttpCaptureHeadersRequest }}'
+            {{- end }}
+            {{- if .HttpCaptureHeadersResponse }}
+            - name: ROUTER_CAPTURE_HTTP_RESPONSE_HEADERS
+              value: '{{ .HttpCaptureHeadersResponse }}'
+            {{- end }}
+            {{- if .HttpCaptureCookies }}
+            - name: ROUTER_CAPTURE_HTTP_COOKIE
+              value: '{{ .HttpCaptureCookies }}'
+            {{- end }}
+            {{- end }}
+            {{- if .HttpErrorCodePages }}
+            - name: ROUTER_ERRORFILE_503
+              value: /var/lib/haproxy/errorfiles/error-page-503.http
+            - name: ROUTER_ERRORFILE_404
+              value: /var/lib/haproxy/errorfiles/error-page-404.http
+            {{- end }}
```

Should we add a comment to `end`s? Everything at the same indent makes it a bit hard to see what ends where.

```suggestion
            {{- end /* HttpErrorCodePages */ }}
```

---

Done